src/main/java/org/graylog2/alarmcallbacks/pagerduty/PagerDutyClient.java

```diff
@@ -154,13 +154,15 @@ private PagerDutyEvent buildPagerDutyEvent(final Stream stream, final AlertCondi
         final String description = "[ " + stream.getTitle() + " ] " + checkResult.getResultDescription() + " - "
                                     + buildStreamLink(clientUrl, stream);
 
+        final List<Message> backlog_items = getAlarmBacklog(checkResult);
```

Comment:
Please use camelCase for variable names, e. g. `backlogItems` and `backlogSummary`.